src/Numerics/DGMethods/DGModel_kernels.jl

```diff
@@ -748,12 +748,20 @@ fluxes, respectively.
             sgeo[_n2, n, f, e⁻],
             sgeo[_n3, n, f, e⁻],
         )
+        bctag = elemtobndy[f, e⁻]
         # Get surface mass, volume mass inverse
         sM, vMI = sgeo[_sM, n, f, e⁻], sgeo[_vMI, n, f, e⁻]
         id⁻, id⁺ = vmap⁻[n, f, e⁻], vmap⁺[n, f, e⁻]
         e⁺ = ((id⁺ - 1) ÷ Np) + 1
 
         vid⁻, vid⁺ = ((id⁻ - 1) % Np) + 1, ((id⁺ - 1) % Np) + 1
+        if bctag != 0
+            # TODO: we will use vmap⁺ to store the boundary element info
+            #be = e⁺
+            #bcid = vid⁺
+            e⁺ = e⁻
+            vid⁺ = vid⁻
+        end
```

Comment:
So eventually all the loads below will be inside this if statement, e.g., it will become an if-else for loading the "plus" side?

Reply:
Possibly, I haven't decided yet.